instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/JdbcInstrumenterFactory.java

```diff
@@ -32,14 +35,30 @@ public final class JdbcInstrumenterFactory {
   private static final JdbcNetworkAttributesGetter netAttributesGetter =
       new JdbcNetworkAttributesGetter();
 
-  public static boolean captureQueryParameters() {
-    return ConfigPropertiesUtil.getBoolean(
-        "otel.instrumentation.jdbc.experimental.capture-query-parameters", false);
+  public static boolean captureQueryParameters(OpenTelemetry openTelemetry) {
+    if (openTelemetry instanceof ExtendedOpenTelemetry) {
+      ExtendedOpenTelemetry extendedOpenTelemetry = (ExtendedOpenTelemetry) openTelemetry;
+
+      DeclarativeConfigProperties instrumentationConfig =
+          extendedOpenTelemetry.getConfigProvider().getInstrumentationConfig();
+
+      if (instrumentationConfig == null) {
+        return false;
+      }
+
+      return instrumentationConfig
```

Comment:
Are there tests for declarative configuration applied to the JDBC instrumentation?